### ISSUE TITLE
Build error on RN 0.47.0-rc.3 (Android)

### ISSUE THREAD

**Issue by romanenko** [react-community/react-native-image-picker#641]
### Description

I've upgraded my project to React 0.47.0-rc.3 and, with `react-native-image-picker` as a dependency it broke Android builds. Build error log:

```
:react-native-image-picker:generateReleaseSources UP-TO-DATE
:react-native-image-picker:incrementalReleaseJavaCompilationSafeguard UP-TO-DATE
:react-native-image-picker:compileReleaseJavaWithJavac
:react-native-image-picker:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/Users/michael/code/exp/Example/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerPackage.java:34: error: method does not override or implement a method from a supertype
  @Override
  ^
1 error
:react-native-image-picker:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-image-picker:compileReleaseJavaWithJavac'.
```

### How to repeat issue and example

`react-native init Example --version 0.47.0-rc.3 && cd Example`
`yarn add react-native-image-picker`
`react-native run-android`

### Solution

I guess, this issue (#307) could be somehow related to this, as the build error look the same.

### Additional Information

* React Native version: 0.47.0-rc.3
* Platform: Android
* Development Operating System: MacOS Sierra 10.12.5
* Dev tools: Android SDK 23

**Comment by rroehrig** [comment]
I still face this problem with the released version 0.47.1.

**Comment by dbanck** [comment]
It might be because [0.47](https://github.com/facebook/react-native/releases/tag/v0.47.) removes unused `createJSModules` calls.

There are several open pull requests fixing this: #619, #649, #653

**Comment by leethree** [comment]
Is there any contributor available to take a look at the PRs?

**Comment by IlyaVi** [comment]
+1

**Comment by janicduplessis** [comment]
Fixed in 0.26.4
